core/src/main/scala/org/apache/spark/deploy/FaultToleranceTest.scala

```diff
@@ -113,7 +112,7 @@ private object FaultToleranceTest extends App with Logging {
     assertValidClusterState()
 
     killLeader()
-    delay(30 seconds)
+    delay(30.seconds)
```

Comment:
yah thats a good point we likely catch them during the codereview process so having the whitelisting probably isn't that bad. I'll switch this to leave the postfix time operators unless someone says otherwise over the weekend.